test/tools/ossfuzz/protomutators/YulProtoMutator.cpp

```diff
@@ -0,0 +1,2081 @@
+#include <test/tools/ossfuzz/protomutators/YulProtoMutator.h>
+
+#include <libyul/Exceptions.h>
+
+#include <src/text_format.h>
+
+using namespace solidity::yul::test::yul_fuzzer;
+
+using namespace protobuf_mutator;
+
+using namespace std;
+
+template <typename Proto>
+using YPR = YulProtoCBRegistration<Proto>;
+using YPM = YulProtoMutator;
+using PrintChanges = YPM::PrintChanges;
+
+MutationInfo::MutationInfo(ProtobufMessage* _message, string const& _info):
+	ScopeGuard([&]{ exitInfo(); }), m_protobufMsg(_message)
```

Comment:
I might be wrong here, but I think if you break the line to start the initialization list you need to break the line for each initializer

Reply:
Updated (break lines)